app/src/main/res/drawable/ic_desktop.xml

```diff
@@ -8,5 +8,5 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
     <path android:fillColor="?primaryText"
-        android:pathData="M4,6a1,1 0,0 1,1 -1h14a1,1 0,0 1,1 1v11h1.5a0.5,0.5 0,0 1,0.5 0.5v1a0.5,0.5 0,0 1,-0.5 0.5h-19a0.5,0.5 0,0 1,-0.5 -0.5v-1a0.5,0.5 0,0 1,0.5 -0.5L4,17L4,6zM10,17v1h4v-1h-4zM6,7v9h12L18,7L6,7z" />
+        android:pathData="M20.5,17h-2a2.5,2.5 0,0 0,2.5 -2.5v-9A2.5,2.5 0,0 0,18.5 3h-13A2.5,2.5 0,0 0,3 5.5v9A2.5,2.5 0,0 0,5.5 17h-2a1.5,1.5 0,0 0,0 3h17a1.5,1.5 0,0 0,0 -3zM5.3,15.5l-0.8,-0.8L4.5,5.3l0.8,-0.8h13.4l0.8,0.8v9.4l-0.8,0.8L5.3,15.5zM13.25,18.5h-2.5a0.75,0.75 0,0 1,0 -1.5h2.5a0.75,0.75 0,0 1,0 1.5z" />
 </vector>
```

Comment:
nit: New line at EOF

Reply:
We could address this in another ticket, as it's not related to this PR (I didn't create the file I updated just the portion on the diff) , and there we can discuss if we want to change that globally.

Reply:
Filed https://github.com/mozilla-mobile/fenix/issues/19273